### PR TITLE
METER-947: rebranding winpcap

### PR DESCRIPTION
**Pull request by kchenhal**
re-branding winpcap, replace boundary with true sight

### CODE REVIEW

File: winpcap-nmap.nsi [hunk 1]
```diff
@@ -160,6 +160,9 @@ Function .onInit
     silent_checks:
       ; check for the presence of Boundary's custom WinPcapInst registry key:
       ReadRegStr $0 "HKLM" "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\WinPcapInst" "InstalledBy"
+	  
+	  ; we are chaning the installed by from Boundary to TrueSight, add un-install logic here
```
Comment (thirty8super) [inline winpcap-nmap.nsi]: "We are changing"....

**Comment by thirty8super** [comment]
+1 with one comment.

**Comment by kchenhal** [comment]
merged #2

**Comment by busterb** [comment]
you guys may want to look at switching to npcap. It is higher performance, and a lot more secure: https://nmap.org/npcap/

**Comment by thirty8super** [comment]
Thanks Brent.  We have a ticket for that currently.  This is our quick catch up on a rebranding "miss".  I blame Pearce ;)

**Pull request merged**